src/parallel/boundaries/BoundaryUtils.cpp

```diff
@@ -0,0 +1,296 @@
+    return DimensionType::ERROR;
+  }
+  switch (findSign(dim)) {
+  case -1:
+    switch (dim) {
+    case 1:
+      return DimensionType::NEGX;
+    case 2:
+      return DimensionType::NEGY;
+    default:
+      return DimensionType::NEGZ; // case 3
+    }
+  case 1:
+    switch (dim) {
+    case 1:
+      return DimensionType::POSX;
+    case 2:
+      return DimensionType::POSY;
+    default:
+      return DimensionType::POSZ; // case 3
+    }
+  default: // should never happen
+    Log::global_log->error()
+        << "Invalid dimension passed for enum conversion" << std::endl;
+    mardyn_exit(1);
+  }
+  return DimensionType::ERROR;
+}
+
+BoundaryUtils::DimensionType
+BoundaryUtils::convertLS1DimsToDimensionPos(int dim) {
+  DimensionType toRet = DimensionType::ERROR;
+  switch (dim) {
+  case 0:
+    toRet = DimensionType::POSX;
+    break;
+  case 1:
+    toRet = DimensionType::POSY;
+    break;
+  default: // case 2:
+    toRet = DimensionType::POSZ;
+  }
+  return toRet;
+}
+
+std::string BoundaryUtils::convertDimensionToString(DimensionType dimension) {
+  switch (dimension) {
+  case DimensionType::POSX:
+    return "+x";
+
+  case DimensionType::POSY:
+    return "+y";
+
+  case DimensionType::POSZ:
+    return "+z";
+
+  case DimensionType::NEGX:
+    return "-x";
+
+  case DimensionType::NEGY:
+    return "-y";
+
+  case DimensionType::NEGZ:
+    return "-z";
+
+  default:
+    Log::global_log->error()
+        << "Invalid dimension passed for enum conversion" << std::endl;
+    mardyn_exit(1);
+    return "error";
+  }
+}
+
+std::string
+BoundaryUtils::convertDimensionToStringAbs(DimensionType dimension) {
+  return convertDimensionToString(dimension).substr(1, 1);
+}
+
+int BoundaryUtils::convertDimensionToNumeric(DimensionType dimension) {
+  switch (dimension) {
+  case DimensionType::POSX:
+    return 1;
+
+  case DimensionType::POSY:
+    return 2;
+
+  case DimensionType::POSZ:
+    return 3;
+
+  case DimensionType::NEGX:
+    return -1;
+
+  case DimensionType::NEGY:
+    return -2;
+
+  case DimensionType::NEGZ:
+    return -3;
+
+  default:
+    Log::global_log->error()
+        << "Invalid dimension passed for enum conversion" << std::endl;
+    mardyn_exit(1);
+    return 0;
+  }
+}
+
+int BoundaryUtils::convertDimensionToNumericAbs(DimensionType dimension) {
+  int toReturn = convertDimensionToNumeric(dimension);
+  return std::abs(toReturn);
+}
+
+int BoundaryUtils::convertDimensionToLS1Dims(DimensionType dimension) {
+  return convertDimensionToNumericAbs(dimension) - 1;
+}
+
+BoundaryUtils::BoundaryType
+BoundaryUtils::convertStringToBoundary(std::string boundary) {
+  if (boundary == "periodic")
+    return BoundaryType::PERIODIC_OR_LOCAL;
+  if (boundary == "reflecting" || boundary == "reflective")
+    return BoundaryType::REFLECTING;
+  if (boundary == "outflow")
+    return BoundaryType::OUTFLOW;
+  Log::global_log->error()
+      << "Invalid boundary type passed. Check your input file!" << std::endl;
```

Comment:
```suggestion
      << "Invalid boundary type passed. Check your input file!" << std::endl;
```
Add what was passed to the output. Same for all other similar error messages.